pkg/router/router.go

```diff
@@ -27,10 +28,11 @@ func (dr *DumbRouter) SetHeaders(w http.ResponseWriter) {
 	h.Set("Server", dr.ServerName)
 }
 
-// RootHandler is a simple http hander for / that returns "Go-Camo"
-func (dr *DumbRouter) RootHandler(w http.ResponseWriter, r *http.Request) {
+// HealthCheckHandler is HTTP handler for confirming the backend service
+// is available from an external client, such as a load balancer.
+func (dr *DumbRouter) HealthCheckHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "text/plain; charset=utf-8")
-	http.Error(w, "404 Not found", 404)
+	io.WriteString(w, dr.ServerName)
```

Comment:
Let's change this to `w.WriteHeader(204) // send the headers with a 204 response code.`
It's probably fine to just return a success status, without returning the server name.

Reply:
Sure! Fixed in 67f7524cd42811d836c14b0d78e6f3fa911f1448